internal/data/spicedb_test.go

```diff
@@ -93,27 +94,27 @@ func TestCreateRelationshipWithSubjectRelation(t *testing.T) {
 
 	container.WaitForQuantizationInterval()
 
-	exists := CheckForRelationship(spiceDbRepo, "bob", "rbac", "user", "", "member", "rbac", "group", "bob_club")
+	exists := CheckForRelationship(spiceDbRepo, "bob", "rbac", "principal", "", "member", "rbac", "group", "bob_club")
 	assert.True(t, exists)
 
 	exists = CheckForRelationship(spiceDbRepo, "bob_club", "rbac", "group", "member", "subject", "rbac", "role_binding", "fan_binding")
 	assert.True(t, exists)
 
 	// zed permission check rbac/role_binding:fan_binding subject rbac/user:bob
```

Comment:
```suggestion
	// zed permission check rbac/role_binding:fan_binding subject rbac/principal:bob
```
Couple comments that still reference user